### PR TITLE
fix: tidy parser. add lint/test to makefile.

### PR DESCRIPTION
**Pull request by dreulavelle**

### CODE REVIEW

File: backend/program/content/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from .mdblist import Mdblist
-from .overseerr import Overseerr
-from .plex_watchlist import PlexWatchlist
-from .listrr import Listrr
\ No newline at end of file
+from .mdblist import Mdblist # noqa: F401
+from .overseerr import Overseerr # noqa: F401
+from .plex_watchlist import PlexWatchlist # noqa: F401
+from .listrr import Listrr # noqa: F401
```
Comment (dreulavelle) [inline backend/program/content/__init__.py:4]: ruff yells at me without the `# noqa`

File: makefile [hunk 2]
```diff
@@ -13,59 +13,39 @@ help:
...
-ec:
-	@docker exec -it iceberg /bin/bash -c "vim /iceberg/data/settings.json"
-
-update:
-	@echo Not implemented yet
-	# @-git pull --rebase
-	# @make start
+lint:
+	ruff check backend
+	pyright backend
```
Comment (dreulavelle) [inline makefile:46]: I'm expecting the lint/test commands to change.. just tossing these here for now.

File: makefile [hunk 3]
```diff
@@ -13,59 +13,39 @@ help:
...
-	@echo frontend  : Start the frontend development server
-	@echo backend   : Start the backend development server
+	@echo lint      : Run the linter and type checker
+	@echo test      : Run the tests
+	@echo pr-ready  : Run the linter and tests
 	@echo -------------------------------------------------------------------------
 
 install:
-	@python3 -m pip install -r requirements.txt --break-system-packages
+	@pip install -r requirements.txt --upgrade --break-system-packages
```
Comment (dreulavelle) [inline makefile:26]: possibly switch to using poetry install? would handle packages better than this :/

**Comment by dreulavelle** [comment]
this ones ready for now

**Review by AyushSehrawat — APPROVED** [review]

**Pull request merged**